packages/hedera/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 0.5.13
+
+### Patch Changes
+
+- Added Hedera module 
+  
```

Comment:
Did you manually create this? The changelog will automatically be created when we make a new release, and be for 0.6.x. I think we can remove the file for now